core/shared/src/main/scala/com/monovore/decline/Parser.scala

```diff
@@ -95,7 +104,7 @@ private[decline] case class Parser[+A](command: Command[A])
         accumulator
           .parseSub(arg)
           .map { result =>
-            result(rest).swap.map { _.withPrefix(List(command.name)) }.swap.flatMap(evalResult)
+            result(rest).leftMap { _.withPrefix(List(command.name)) }.flatMap(evalResult)
```

Comment:
Good call... I think this dates back to an older Scala version where this API was unavailable.